tests/test_integration/test_toolkit_client/test_search_config.py

```diff
@@ -0,0 +1,73 @@
+from random import randint
+
+import pytest
+
+from cognite_toolkit._cdf_tk.client import ToolkitClient
+from cognite_toolkit._cdf_tk.client.data_classes.search_config import (
+    SearchConfig,
+    SearchConfigList,
+    SearchConfigView,
+    SearchConfigWrite,
+)
+
+SEARCH_CONFIG_NAME = "Search Config Name"
+
+
+@pytest.fixture(scope="session")
+def existing_search_config(toolkit_client: ToolkitClient) -> dict:
```

Comment:
Why not return a `SearchConfig` here?
```suggestion
def existing_search_config(toolkit_client: ToolkitClient) -> SearchConfig:
```

Reply:
I also wanted count of already present configs thus was returning in dict. Could use tuple instead. with first element as count and second as SearchConfig. id can be fetched using SearchConfig.id